source/main/utils/buildAppMenus.js

```diff
@@ -49,8 +55,34 @@ export const buildAppMenus = async (
     safeExitWithCode(22);
   };
 
-  const { isMacOS } = environment;
+  const {
+    isMacOS,
+    version,
+    apiVersion,
+    network,
+    build,
+    installerVersion,
+    os,
+    buildNumber,
+  } = environment;
+
   const translations = require(`../locales/${locale}`);
+
+  const networkLocale = getLocale(network);
+
+  const supportRequestData = {
```

Comment:
@DeeJayElly please see if we can use a generic solution where we define the support request payload and url in common and use both in main and rendering process 🙏

Reply:
@DeeJayElly any news on this one?

Reply:
Didn't found any suitable solution. I have a feeling that we should leave it like this for now, if you agree @nikolaglumac

Reply:
OK. I have added a card for that so that we improve it in the future: https://iohk.myjetbrains.com/youtrack/issue/DDW-689